bugbug/models/duplicate.py

```diff
@@ -61,9 +61,11 @@ def __init__(self, lemmatization=False):
 
     def get_labels(self):
         bugs = []
+        reporters = ("intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs")
 
         for bug_data in islice(bugzilla.get_bugs(), 0, NUM_DUPLICATES):
-            bugs.append(bug_data["id"])
+            if bug_data["creator"] not in reporters:
```

Comment:
There is another place in get_labels where you are iterating through bugs, we should ignore these reporters there too